src/fenicsxconcrete/experimental_setup/compression_cylinder.py

```diff
@@ -191,49 +219,96 @@ def create_displacement_boundary(self, V:df.fem.FunctionSpace) -> list:
         # define boundary conditions generator
         bc_generator = BoundaryConditions(self.mesh, V)
 
-        if self.p['bc_setting'] == "fixed":
-            if self.p['dim'] == 2:
-                bc_generator.add_dirichlet_bc(self.top_displacement, self.boundary_top(), 1, "geometrical", 1)
-                bc_generator.add_dirichlet_bc(np.float64(0.0), self.boundary_top(), 0, "geometrical", 0)
-                bc_generator.add_dirichlet_bc(df.fem.Constant(domain=self.mesh, c=(0.0, 0.0)), self.boundary_bottom(), None, "geometrical")
-            elif self.p['dim']== 3:
-
-                bc_generator.add_dirichlet_bc(self.top_displacement, self.boundary_top(), 2, "geometrical", 2)
-                bc_generator.add_dirichlet_bc(np.float64(0.0), self.boundary_top(), 0, "geometrical", 0)
-                bc_generator.add_dirichlet_bc(np.float64(0.0), self.boundary_top(), 1, "geometrical", 1)
-                bc_generator.add_dirichlet_bc(df.fem.Constant(domain=self.mesh, c=(0.0, 0.0, 0.0)), self.boundary_bottom(), None, "geometrical")
-
-        elif self.p['bc_setting'] == "free":
-            if self.p['dim'] == 2:
-                bc_generator.add_dirichlet_bc(self.top_displacement, self.boundary_top(), 1, "geometrical", 1)
-                bc_generator.add_dirichlet_bc(np.float64(0.0), self.boundary_bottom(), 1, "geometrical", 1)
-                bc_generator.add_dirichlet_bc(np.float64(0.0), point_at((0, 0)), 0, "geometrical", 0)
-
-            elif self.p['dim'] == 3:
+        if self.p["bc_setting"] == "fixed":
+            if self.p["dim"] == 2:
+                bc_generator.add_dirichlet_bc(
+                    self.top_displacement, self.boundary_top(), 1, "geometrical", 1
+                )
+                bc_generator.add_dirichlet_bc(
+                    np.float64(0.0), self.boundary_top(), 0, "geometrical", 0
+                )
+                bc_generator.add_dirichlet_bc(
+                    df.fem.Constant(domain=self.mesh, c=(0.0, 0.0)),
+                    self.boundary_bottom(),
+                    None,
+                    "geometrical",
+                )
+            elif self.p["dim"] == 3:
+                bc_generator.add_dirichlet_bc(
+                    self.top_displacement, self.boundary_top(), 2, "geometrical", 2
+                )
+                bc_generator.add_dirichlet_bc(
+                    np.float64(0.0), self.boundary_top(), 0, "geometrical", 0
+                )
+                bc_generator.add_dirichlet_bc(
+                    np.float64(0.0), self.boundary_top(), 1, "geometrical", 1
+                )
+                bc_generator.add_dirichlet_bc(
+                    df.fem.Constant(domain=self.mesh, c=(0.0, 0.0, 0.0)),
+                    self.boundary_bottom(),
+                    None,
+                    "geometrical",
+                )
```

Comment:
@aradermacher
In this case I dont this it is an imporvement or gives a better overview.
are there ways to configure this, or is the whole point, that the formating is fixed automatically?
Is it just the character limit per line that gives this result?

Reply:
Probably, I would also be in favor of increasing the character limit to let's say 120?! Especially, if we have descriptive variable names, 80 characters can run out pretty fast. There should be a way to configure black for that purpose.

Reply:
@eriktamsen I agree that sometimes the black foramtting is not that intuitive (that' why my editor is doing it for me :) )

@srosenbu Yes that's possible: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html